refactor: fix mypy errors for checkers/b*

Fix mypy errors for the following:

```
cve_bin_tool/checkers/bubblewrap.py:15: error: Need type annotation for "CONTAINS_PATTERNS" (hint: "CONTAINS_PATTERNS: List[<type>] = ...")
cve_bin_tool/checkers/bison.py:15: error: Need type annotation for "CONTAINS_PATTERNS" (hint: "CONTAINS_PATTERNS: List[<type>] = ...")
cve_bin_tool/checkers/bison.py:16: error: Need type annotation for "FILENAME_PATTERNS" (hint: "FILENAME_PATTERNS: List[<type>] = ...")
cve_bin_tool/checkers/bird.py:16: error: Need type annotation for "CONTAINS_PATTERNS" (hint: "CONTAINS_PATTERNS: List[<type>] = ...")
cve_bin_tool/checkers/bird.py:17: error: Need type annotation for "FILENAME_PATTERNS" (hint: "FILENAME_PATTERNS: List[<type>] = ...")

```

Helpful resources:

- [Python typing documentation](https://docs.python.org/3/library/typing.html)
- [mypy cheat sheet](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html)
- If you can't find the answer in the docs, googling the error messages above will often help!
- [cve-bin-tool's contributor docs](https://github.com/intel/cve-bin-tool/blob/main/CONTRIBUTING.md)
- If you've contributed to open source but not this project, you might just want our [checklist for a great pull request](https://github.com/intel/cve-bin-tool/blob/main/CONTRIBUTING.md#checklist-for-a-great-pull-request)

Tiny tips for new contributors:

- cve-bin-tool uses https://www.conventionalcommits.org/ style for commit messages, and we have a test that checks the title of your PR. A good potential title for this one is in the title of this issue.
- You can make an issue auto close by including a comment "fixes #ISSUENUMBER" in your PR comments where ISSUENUMBER is the actual number of the issue.

--- a/cve_bin_tool/checkers/bird.py
+++ b/cve_bin_tool/checkers/bird.py
@@ -9,11 +9,13 @@ https://www.cvedetails.com/product/68129/Bird-Project-Bird.html?vendor_id=21737
 https://www.cvedetails.com/product/59242/NIC-Bird.html?vendor_id=20536
 
 """
+from typing import List
+
 from cve_bin_tool.checkers import Checker
 
 
 class BirdChecker(Checker):
-    CONTAINS_PATTERNS = []
-    FILENAME_PATTERNS = []
+    CONTAINS_PATTERNS: List[str] = []
+    FILENAME_PATTERNS: List[str] = []
     VERSION_PATTERNS = [r"BIRD ([0-9]+\.[0-9]+\.[0-9]+)"]
     VENDOR_PRODUCT = [("bird_project", "bird"), ("nic", "bird")]

--- a/cve_bin_tool/checkers/bison.py
+++ b/cve_bin_tool/checkers/bison.py
@@ -8,11 +8,13 @@ CVE checker for bison
 https://www.cvedetails.com/product/86839/GNU-Bison.html?vendor_id=72
 
 """
+from typing import List
+
 from cve_bin_tool.checkers import Checker
 
 
 class BisonChecker(Checker):
-    CONTAINS_PATTERNS = []
-    FILENAME_PATTERNS = []
+    CONTAINS_PATTERNS: List[str] = []
+    FILENAME_PATTERNS: List[str] = []
     VERSION_PATTERNS = [r"Bison ([0-9]+\.[0-9]+\.[0-9]+)"]
     VENDOR_PRODUCT = [("gnu", "bison")]

--- a/cve_bin_tool/checkers/boinc.py
+++ b/cve_bin_tool/checkers/boinc.py
@@ -9,12 +9,14 @@ https://www.cvedetails.com/product/27779/Rom-Walton-Boinc.html?vendor_id=13367
 https://www.cvedetails.com/product/63697/Berkeley-Boinc.html?vendor_id=356
 
 """
+from typing import List
+
 from cve_bin_tool.checkers import Checker
 
 
 class BoincChecker(Checker):
-    CONTAINS_PATTERNS = []
-    FILENAME_PATTERNS = []
+    CONTAINS_PATTERNS: List[str] = []
+    FILENAME_PATTERNS: List[str] = []
     VERSION_PATTERNS = [
         r"boinc.so.([0-9]+\.[0-9]+\.[0-9]+)",
         r"([0-9]+\.[0-9]+\.[0-9]+)\r?\nboinc",

--- a/cve_bin_tool/checkers/bro.py
+++ b/cve_bin_tool/checkers/bro.py
@@ -8,11 +8,13 @@ CVE checker for bro
 https://www.cvedetails.com/product/37247/BRO-BRO.html?vendor_id=16374
 
 """
+from typing import List
+
 from cve_bin_tool.checkers import Checker
 
 
 class BroChecker(Checker):
-    CONTAINS_PATTERNS = []
-    FILENAME_PATTERNS = []
+    CONTAINS_PATTERNS: List[str] = []
+    FILENAME_PATTERNS: List[str] = []
     VERSION_PATTERNS = [r"bro-([0-9]+\.[0-9]+\.?[0-9]*)"]
     VENDOR_PRODUCT = [("bro", "bro")]

--- a/cve_bin_tool/checkers/bubblewrap.py
+++ b/cve_bin_tool/checkers/bubblewrap.py
@@ -8,11 +8,13 @@ CVE checker for bubblewrap
 https://www.cvedetails.com/vendor/16078/?q=Bubblewrap+Project
 
 """
+from typing import List
+
 from cve_bin_tool.checkers import Checker
 
 
 class BubblewrapChecker(Checker):
-    CONTAINS_PATTERNS = []
+    CONTAINS_PATTERNS: List[str] = []
     FILENAME_PATTERNS = [r"bwrap"]
     VERSION_PATTERNS = [r"bubblewrap ([0-9]+\.[0-9]+\.[0-9]+)"]
     VENDOR_PRODUCT = [

